test/basic.test.ts

```diff
@@ -96,7 +97,7 @@ test('it should throw an error if building without an outdir or outfile', async
   try {
     await require('esbuild').build(buildOptions({}, {outdir: undefined, outfile: undefined}));
   } catch (e) {
-    expect(e.message).toMatch(/outdir/);
+    expect(e.message).toMatch(/output/);
```

Comment:
Any particular reason this needed to be changed?

Reply:
I'm not sure, the test failed when testing earlier, but after I refactored the tests and added my own tests for this code, I had to revert it again :)